be/src/service/service_be/starrocks_be.cpp

```diff
@@ -180,7 +181,7 @@ void start_be(const std::vector<StorePath>& paths, bool as_cn) {
         LOG(ERROR) << "Fail to init datacache";
         exit(1);
     }
-    LOG(INFO) << "BE start step " << start_step++ << ": datacache init successfully";
+    LOG_IF(INFO, config::datacache_enable) << "BE start step " << start_step++ << ": datacache init successfully";
```

Comment:
may still always log this line, but the content can be `init successful` or `init skipped`

Reply:
> may still always log this line, but the content can be `init successful` or `init skipped`

Changed